Source/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ if(${UNIVERSAL_BUILD})
     endif()
     include(cmake_core.cmake)
 else()
-    set(ARTEFACTS none neon avx2 sse4.1 sse2)
-    set(CONFIGS ${ISA_NONE} ${ISA_NEON} ${ISA_AVX2} ${ISA_SSE41} ${ISA_SSE2})
+    set(ARTEFACTS native none neon avx2 sse4.1 sse2)
```

Comment:
There is one list like this in `Source/UnitTest/CMakeLists.txt` as well. Is that intentionally left out?

Reply:
Good spot.